tests/integration/graph/test_graph_responsive.py

```diff
@@ -0,0 +1,125 @@
+import pytest
+import pandas as pd
+from multiprocessing import Value
+import numpy as np
+from time import sleep
+
+import dash
+import dash_html_components as html
+import dash_core_components as dcc
+from dash.dependencies import Input, Output, State
+from dash.exceptions import PreventUpdate
+from utils import wait_for
+
+
+@pytest.mark.parametrize("responsive", [True, False, None])
+@pytest.mark.parametrize("autoresize", [True, False, None])
+@pytest.mark.parametrize("height", [600, None])
+@pytest.mark.parametrize("width", [600, None])
+@pytest.mark.parametrize("is_responsive", [True, False, 'auto'])
+def test_grrs001_graph(
```

Comment:
This test makes sure that all combinations of responsiveness / dimensions behave as intended.
Not taking any screenshot, we're just interested in the size of the graph div on 1st render and on resize.